tools/lsp/preview/native.rs

```diff
@@ -154,6 +154,7 @@ fn open_ui_impl(preview_state: &mut PreviewState) {
         slint::CloseRequestResponse::HideWindow
     });
     ui.show().unwrap();
+    ui.window().set_size(slint::PhysicalSize { width: 800, height: 600 });
```

Comment:
This should be LogicalSize.

But I'd say that it shouldn't be set from here, but as preferred size in the slint code.